Kateryna_Volynets/Paired_and_unpaired.py

```diff
@@ -0,0 +1,15 @@
+numbers = [1, 2, 3, 4, 5, 6, 7, 8]
+paired_value, unpaired_value = [], []
+paired_index, unpaired_index = [], []
+for index, i in enumerate(numbers):
+    if index % 2 == 1:
+        unpaired_index.append(index)
+        unpaired_value.append(i)
+    else:
+        paired_index.append(index)
+        paired_value.append(i)
+

```

Comment:
over-engineering. 
you can just have two lists pair_number and unpair_number

and in if statement you can use pair_number.apend((index, number))